lib/Scheduler/aco.cpp

```diff
@@ -398,6 +424,11 @@ FUNC_RESULT ACOScheduler::FindSchedule(InstSchedule *schedule_out,
     writePheromoneGraph("iteration" + std::to_string(iterations));
     iterations++;
   }
+
+  Logger::Event(IsPostBB ? "ACOpost_sched_complete" : "ACO_sched_complete",
+                "cost", bestSchedule->GetCost(),
+                "iterations", iterations,
+                "improvement", InitialCost-bestSchedule->GetCost());
```

Comment:
The other `Logger::Event(...)` calls would format the event_id as `AcoSchedComplete` or `ACOSchedComplete`.

Another thing to note: we could do this conditional event_id. On the other hand, we could just do `ACOSchedComplete` with another parameter like `"pass", IsPostBB ? "normal" : "post_bb"`. I don't know which is the better strategy.

Reply:
I can change the event ID format, but adding a pass parameter would be a bad idea since there can be both a preBB and postBB run of the ACO Scheduler